specs/Qowaiv.Specs/Text/Wild_card_pattern_specs.cs

```diff
@@ -4,7 +4,97 @@ public class Is_match
 {
     [Test]
     public void default_without_specials_for_current_culture()
-        =>  WildcardPattern.IsMatch("%", "a").Should().BeFalse();
+        => WildcardPattern.IsMatch("%", "a").Should().BeFalse();
+
+    [TestCase("*", "matches")]
+    [TestCase("转*字", "转注字")]
+    [TestCase("转?字", "转注字")]
+    [TestCase("g*ks", "geeks")]
+    [TestCase("g*éks", "geéks")]
+    [TestCase("G*ks", "Geeks")]
+    [TestCase("ge?ks*", "geeksforgeeks")]
+    [TestCase("abc*bcd", "abcdhghgbcd")]
+    [TestCase("*c*d", "abcd")]
+    [TestCase("*?c*d", "abcd")]
+    public void using_wildcards(string pattern, string input)
+        => WildcardPattern.IsMatch(pattern, input)
+        .Should().BeTrue();
+
+    [TestCase("转%字", "转注字")]
+    [TestCase("转_字", "转注字")]
+    [TestCase("g%ks", "geeks")]
+    [TestCase("g%éks", "geéks")]
+    [TestCase("G%ks", "Geeks")]
+    [TestCase("ge_ks%", "geeksforgeeks")]
+    [TestCase("abc%bcd", "abcdhghgbcd")]
+    [TestCase("%c%d", "abcd")]
+    [TestCase("%_c%d", "abcd")]
+    public void using_SQL_wildcards(string pattern, string input)
+        => WildcardPattern.IsMatch(pattern, input, WildcardPatternOptions.SqlWildcards, StringComparison.Ordinal)
+        .Should().BeTrue();
+
+    [TestCase("gee?ks", "geeks")]
+    [TestCase("ge?ks", "geeks")]
+    [TestCase("Qowaiv?", "Qowaiv")]
+    [TestCase("Qowaiv??", "Qowaiv")]
+    [TestCase("Qowaiv*", "Qowaiv")]
+    [TestCase("Qowaiv*?*", "Qowaiv")]
+    public void with_trailing(string pattern, string input) 
+        => WildcardPattern.IsMatch(pattern, input, WildcardPatternOptions.SingleOrTrailing, StringComparison.Ordinal)
+        .Should().BeTrue();
+
+    [TestCase("gee_ks", "geeks", true, WildcardPatternOptions.SingleOrTrailing | WildcardPatternOptions.SqlWildcards)]
```

Comment:
The `gee_ks` matches `geeks` here because of the ` WildcardPatternOptions.SingleOrTrailing`? Because I can't say that's what I would think of when seeing `Trailing`.